paasta_tools/kubernetes_tools.py

```diff
@@ -177,7 +177,7 @@
 filters = filter('paasta_service', '{paasta_service}') and filter('paasta_instance', '{paasta_instance}') and filter('paasta_cluster', '{paasta_cluster}')
 
 current_replicas = data('kube_hpa_status_current_replicas', filter=filters, extrapolation="last_value").sum(by=['paasta_cluster'])
```

Comment:
We'd have to think through the implications, but I wonder if this should just be the count of load timeseries? Since we divide the summed load by this number, I think that might give us a more accurate average.

Reply:
I think maybe the issue there would be that the `uwsgi` timeseries is only collected for pods that have passed their readiness check, but we need to provide a scaling target for the total number of pods for the Deployment, so it might lead to us overscaling?